### ISSUE TITLE
invalid signature: 0x80014

### ISSUE THREAD

**Issue by cricri** [guyonroche/exceljs#87]
Hi,

I can reproduce the "Too Many Worksheets Results in Parse Error" bug. In  my case, it's not the number of sheets which cause the bug, but rather the "quantity of data".
This script generates a xlsx with 3 sheets of 200 lines x 24 columns but cannot parse it.

fedora 23/exceljs@0.2.4/unzip@0.1.11

```js
var Excel = require("exceljs");

var filename = '/tmp/bug.xlsx';

var workbook1 = new Excel.Workbook();
for (var i=0; i<3; i++) {
  var sheet = workbook1.addWorksheet("Sheet "+i);
  for (var j=0; j<200; j++)
    sheet.addRow([1,2,3,4,5,6,7,8,9,10,11,12,1,2,3,4,5,6,7,8,9,10,11,12]);
}

workbook1.xlsx.writeFile(filename)
  .then(function() {
    var workbook2 = new Excel.Workbook();
    workbook2.xlsx.readFile(filename)
      .then(console.log);
  });
```

**Comment by guyonroche** [comment]
I'm in the process of merging (and testing) [fix #28 invalid signature: 0x80014](https://github.com/guyonroche/exceljs/pull/51)
Will close this soon.

**Comment by guyonroche** [comment]
Fixed in 0.2.6
